src/index.html

```diff
@@ -5,8 +5,361 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img
+          class="header__logo-img"
+          src="images\logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--is-active is-active" href="/">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/">
+              Smartphones
+            </a>
+          </li>
+          <li data-qa="nav-hover" class="nav__item">
+            <a class="nav__link" href="/">
+              Laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/">
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="grid-container">
+      <div class="card" data-qa="card">
+
+        <img
+          src="./images/imac.jpeg"
+          alt="product image"
+          class="card__img"
+        >
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating-block">
+
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rewiews-number">Reviews: 5</p>
+
+        </div>
+
+        <div class="card__price-block">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <button class="card__button" data-qa="card-hover">Buy</button>
+
+      </div>
```

Comment:
No extra new line between first and last sibling tag.
Fix all similar cases
```suggestion
        <button class="card__button" data-qa="card-hover">Buy</button>
      </div>
```